lib/node_modules/@stdlib/stats/base/mskmax/benchmark/benchmark.ndarray.js

```diff
@@ -21,13 +21,24 @@
 // MODULES //
 
 var bench = require( '@stdlib/bench' );
-var randu = require( '@stdlib/random/base/randu' );
 var isnan = require( '@stdlib/math/base/assert/is-nan' );
+var uniform = require( '@stdlib/random/array/uniform' );
+var bernoulli = require( '@stdlib/random/array/bernoulli' );
 var pow = require( '@stdlib/math/base/special/pow' );
 var pkg = require( './../package.json' ).name;
 var mskmax = require( './../lib/ndarray.js' );
 
 
+// VARIABLES //
+
+var uniformOptions = {
+	'dtype': 'generic'
+};
+var bernoulliOptions = {
+	'dtype': 'uint8'
```

Comment:
```suggestion
	'dtype': 'generic'
```

Preserve the dtypes of the original arrays.